src/libs/ajax.js

```diff
@@ -95,16 +95,16 @@ export const DAC = {
     return res.json();
   },
 
-  create: async (name, description) => {
+  create: async (name, description, email) => {
     const url = `${await getApiUrl()}/api/dac`;
-    const dac = { 'name': name, 'description': description };
+    const dac = { 'name': name, 'description': description, 'email': email };
```

Comment:
you can do the same thing as below here.  E.g.
```suggestion
    const dac = { name, description, email };
```